### ISSUE TITLE
Issues with RF run

### ISSUE THREAD

**Issue by jaktykusuma** [diyabc/diyabcGUI#112]
Bonjour,

I had issues with these following message when I tried to run the Random Forest:
   
`  
dyld: lazy symbol binding failed: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/jakty/Library/R/3.6/library/diyabcGUI/bin/abcranger-macos-v1.2.64 (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: ____chkstk_darwin
  Referenced from: /Users/jakty/Library/R/3.6/library/diyabcGUI/bin/abcranger-macos-v1.2.64 (which was built for Mac OS X 10.15)
  Expected in: /usr/lib/libSystem.B.dylib

`

Let me know how to fix this. 
Thanks

**Comment by gdurif** [comment]
Hi,

Thanks for the report. Sorry for my late reply. I am buried in work at the moment. We have a big problem with the latest release (that I downgraded to pre-release) at the moment. I will keep you updated when I have any news.

I will investigated this point also.

Best

**Comment by gdurif** [comment]
Could you try with the latest pre-release (v1.1.1-beta) ? Thanks

**Comment by jaktykusuma** [comment]
Hello,

Many thanks for the updates. However, I've tried running the analysis with the latest pre-release as you suggested, but still generates the same error. 
![Screen Shot 2021-11-30 at 16 26 07 (2)](https://user-images.githubusercontent.com/63284031/144076380-1071a08c-2b16-4eb7-ac28-90f300b2c238.png)

I tried to use my friend's Windows computer, it didn't generate errors no matter the version I used.

Your help is kindly appreciated.

**Comment by gdurif** [comment]
It seems that the problem comes from the [`abcranger`](https://github.com/diyabc/abcranger) program that the app runs in backend.

I opened an issue there (c.f. https://github.com/diyabc/abcranger/issues/90)

Could you give me more details about your OS version ?

**Comment by jaktykusuma** [comment]
Bonjour,

I have found the problem. It seems that OS version lower than 10.13 would not work to run the RF analysis. I have updated my OS to 10.14, and it worked really well. 
![Screen Shot 2021-12-01 at 16 00 42 (2)](https://user-images.githubusercontent.com/63284031/144258255-fca5a80a-a27a-45b7-b021-51cf7a9abffd.png)

Many thanks for your effort in fixing this. Really appreciate it. My own problem now to fix the parameter estimation, strange output value though.

Have a nice evening.
